scripts/build

```diff
@@ -67,6 +67,10 @@ TAGS="ctrd apparmor seccomp netcgo osusergo providerless urfave_cli_no_docs"
 RUNC_TAGS="apparmor seccomp"
 RUNC_STATIC="static"
 
+if [ ${OS} = windows ]; then
+    TAGS="netcgo osusergo providerless"
```

Comment:
Why does Windows use different TAGS? ctrd MUST be included in TAGS to make the built-in version of containerd work.